app/controllers/comments_controller.rb

```diff
@@ -37,6 +44,71 @@ def destroy
                 )
   end
 
+  def vote
```

Comment:
If you can try to clean his method a bit, that would be nice, I know you just moved it, but it would be cleaner (for example, why do we have a rescue, maybe some stuff can be merged together to make it clearer etc.)